### PR TITLE
ADDED:terraform state and variables added

### PR DESCRIPTION
**Pull request by 582Bamfo**
Boss, can you please have a quick look at the variables and the backend file

### CODE REVIEW

**Comment by Nanadennis** [comment]
Good job !! go ahead and merge

File: main.tf [hunk 1]
```diff
@@ -2,23 +2,30 @@ terraform {
...
+    key    = "backend/dev/jay/terraformstate"
+    region = "eu-west-2"
+  }
 }
 
+
+
 provider "aws" {
-  region = "eu-west-2"
+  region = var.region
```
Comment (WiLDglobalTech) [inline main.tf:18]: I cannot find the variable file

**Comment by weajay** [comment]
Looks good, thank you!

**Comment by kwadwo-appiah** [comment]
Hello, buddy is free from conflict. go ahead and merge

**Review by WiLDglobalTech — COMMENTED** [review]
Where is the variable file

**Comment by Johnofosu** [comment]
very good code go ahead deploy it

File: main.tf (hunk 1 above)
Comment (582Bamfo) [reply main.tf:18]: it is the variable block

Comment (WiLDglobalTech) [reply main.tf:18]: Seen Good to go

**Review by WiLDglobalTech — APPROVED** [review]
LGTM

**Pull request merged**